src/__tests__/visual/_page-objects/login.pageobject.ts

```diff
@@ -0,0 +1,27 @@
+import { Page } from '@playwright/test';
+
+export class LoginPage {
+  constructor(public readonly page: Page) {}
+
+  async open() {
+    await this.page.goto('/');
+
+    await this.page.waitForSelector('form#login-form');
+  }
+
+  async login(xprivKey?: string) {
```

Comment:
It should be `loginAsAdmin()` because you're logging in as admin not user.

Reply:
Yup. I will change this.